testing/unit/tp/node_account/shared.py

```diff
@@ -0,0 +1,56 @@
+from sawtooth_sdk.protobuf.setting_pb2 import Setting
+
+from testing.mocks.stub import StubContext
+from remme.tp.node_account import NodeAccountHandler, NodeAccount
+from remme.settings import SETTINGS_MINIMUM_STAKE
+from remme.settings.helper import _make_settings_key
+
+RANDOM_NODE_PUBLIC_KEY = '039d6881f0a71d05659e1f40b443684b93c7b7c504ea23ea8949ef5216a2236940'
+
+MINIMUM_STAKE = 250000
+
+ACCOUNT_ADDRESS_FROM = '116829d71fa7e120c60fb392a64fd69de891a60c667d9ea9e5d9d9d617263be6c20202'
```

Comment:
If you write tests for node account, use `node account` anywhere, because tests should shout about use cases.

In storing public keys we use word `certificate` because it is a use case, despite the fact that node does not know about certificates.

Reply:
Done. Renamed to NODE_ACCOUNT_ADDRESS_FROM.